kotoed-server/src/main/kotlin/org/jetbrains/research/kotoed/api/SubmissionCodeVerticle.kt

```diff
@@ -252,64 +267,14 @@ class SubmissionCodeVerticle : AbstractKotoedVerticle() {
                         revision = repoInfo.revision
                 )
         )
-        val diff = submissionCodeDiff(submission, repoInfo)
         return ListResponse(
                 root = buildCodeTree(
-                        innerResp.files,
-                        diff.contents.flatMap { listOf(it.fromFile, it.toFile) }
```

Comment:
We do not return diff from here anymore, `handleSubmissionCodeDiff` is now used everywhere.